torch/testing/_internal/common_optimizers.py

```diff
@@ -1487,6 +1514,48 @@ def _get_optim_inputs_including_global_cliquey_kwargs(
                 "TestOptimRenewed",
                 "test_deepcopy_copies_all_public_attrs",
             ),
+            DecorateInfo(
+                skipIfTorchDynamo(
+                    "See https://github.com/pytorch/pytorch/issues/115607"
+                ),
+                "TestOptimRenewed",
+                "test_foreach_matches_forloop",
+            ),
+            DecorateInfo(
+                toleranceOverride(
+                    {
+                        # previously atol=1e-7, rtol=1e-7
+                        torch.float64: tol(atol=1.5e-7, rtol=1.1e-7)
```

Comment:
I don't like this...do you have an explanation why?

Reply:
not really, other than there are a lot of ops in RAdam capturable so I think there are most casts vs for loop. These are really close I don't think there's an issue.

Reply:
We should check if this is still applicable once the single tensor capturable happens.